lib/activerecord-import/import.rb

```diff
@@ -335,18 +335,16 @@ def import_helper( *args )
         end
 
         array_of_attributes = models.map do |model|
-            # this next line breaks sqlite.so with a segmentation fault
-            # if model.new_record? || options[:on_duplicate_key_update]
-            column_names.map do |name|
-              name = name.to_s
-              if respond_to?(:defined_enums) && defined_enums.key?(name) # ActiveRecord 5
-                model.read_attribute(name)
-              elsif model.class.column_defaults[name].is_a?(Integer)
```

Comment:
@zdennis sorry I didn't realize there wasn't a test added for that specific issue. That commit fixed the issue when an enum attribute had a default value set on the database column. I've added a test case for that.